_docs/crd-migration-guide.md

```diff
@@ -0,0 +1,68 @@
+# CRDs migration guide for provider from v0.14.x to TBD
+
+[Version](#version) will introduce changes in CRDs that need to be applied in order to continue operations.
+
+All steps below MUST be executed in same order as declared and applies to your environment.
+If any step fails STOP and reach out to Akash team
+
+## Provider service is running on bare-metal server
+1. Stop provider service. Most probably it is running by `systemd`
+2. Upgrade RPC node to [Version](#version). If you are using 3rd-party RPC node ensure upgrade has been applied
+3. Download and install **link to release**
+4. Install new CRDs
+```shell
+kubectl apply -f https://raw.githubusercontent.com/ovrclk/akash/v0.16.0/pkg/apis/akash.network/crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/ovrclk/akash/v0.16.0/pkg/apis/akash.network/provider_hosts_crd.yaml
+``` 
+5. Perform migration
+```shell
+akash provider migrate crds
+```
+6. Start provider service
+
+## Provider service is running in kubernetes
+1. Stop provider service. Scale provider deployment to `0`
+2. Upgrade RPC node to [Version](#version). If you are using 3rd-party RPC node ensure upgrade has been applied
+3. Prepare kubeconfig for cluster in either ~/.kube/config or have `KUBECONFIG` environment variable set. **Skip this step if provider service running under `systemd` on bare-metal.**
```

Comment:
This section says 'running in kubernetes' so maybe the bold text is  not needed?

Reply:
i'll be removing this guide in favor of one in google docs